Unix/scripts/installntlm-ubuntu16

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+
+OMI_DEB=$(readlink -f ~/omi-1.1.0-2.rhel.x64.deb)
```

Comment:
readlink doesn't work on UNIX platforms; I guess since this is specific to Ubuntu 16 (based on it's name), that doesn't matter?

We have other scripts that do this sort of thing in a more transparent way, just FYI ...